configs/sites/aws-pcluster/README.md

```diff
@@ -0,0 +1,178 @@
+## spack-stack on AWS parallelcluster using Intel+Intel-MPI
+
+### Base instance
+- AMI ID: ami-091017c7508ac95f6
+- Instance c5n.4xlarge
+
+### Prerequisites
+1. As `root`:
```

Comment:
Are these instructions intended for anyone, and if so would anyone have root access on the instance?

I might be missing a part of the process. Is the idea that you initially build an EC2 instance (Base instance info on lines 4, 5), log onto that instance and follow instructions to create the stack (lines 7 and onward). Then later you do something to make this run as a parallelcluster.

Or is it the fact that you selected a `c5n.4xlarge` instance that says its a parallelcluster from the start.

Perhaps some discussion at the beginning of this README.md file could help clarify this. Thanks!

Reply:
That's right. I should add to the top of this README that this set of instructions was used to CREATE the site config aws-pcluster. If you look at the readthedocs instructions, the maintainers section refers to the readme for creating the site config. The placeholders in the platforms section will simply say "in order to use ..." launch AMI (the pre-installed AMI that we provide - i.e. based on the basic pcluster + spack-stack release) using instance type xyz.